### PR TITLE
Backport changes to make daily build work

### PR DESCRIPTION
**Pull request by brian-avery**
Backport changes from daily build to operator.

### CODE REVIEW

File: build/generate-manifests.sh [hunk 1]
```diff
@@ -56,13 +56,20 @@ function generateCRDs() {
...
 
   cat ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml >deploy/src/crd.yaml
   echo -e "\n---\n" >>deploy/src/crd.yaml
   cat ${BUNDLE_DIR}/servicemeshmemberrolls.crd.yaml >>deploy/src/crd.yaml
   echo -e "\n---\n" >>deploy/src/crd.yaml
   cat ${BUNDLE_DIR}/servicemeshmembers.crd.yaml >>deploy/src/crd.yaml
+
+  currentDir=$(pwd)
+  cd ${BUNDLE_DIR}
+  sed -i -e '/x-kubernetes-list-map-keys:/,/x-kubernetes-list-type: map/ { /- protocol/d }' *.crd.yaml
```
Comment (rcernich) [inline build/generate-manifests.sh:70]: Can you move this to the original fix, which was on line 53?

Comment (brian-avery) [reply build/generate-manifests.sh:70]: Sure. I thought that was a separate fix since this doesn't modify default values or descriptions.

**Comment by brian-avery** [comment]
Aargh. Problem working with the same branch name in my remote. @jwendell @rcernich Should I revert or would you just like to review as is? I removed the code from line 53 that removes defaults.

File: build/generate-manifests.sh (hunk 1 above)
Comment (rcernich) [reply build/generate-manifests.sh:70]: It fixes the same problem, but it only works on 4.5+.  Having the comment too is a hint about what's going on so we can remove it once we're no longer supporting 4.4.

**Comment by rcernich** [comment]
Just create a new pr.

**Pull request merged**